COMETwebapp/COMETwebapp/Pages/Index.razor

```diff
@@ -0,0 +1,6 @@
+﻿@page "/"
```

Comment:
Please add File header. Maybe discuss with @samatrhea what it should look like in Blazor/Razor.